docs/con_functions.html

```diff
@@ -124,7 +124,19 @@ <h3 class="title"><a href="index.html">OpenShift Serverless Previews</a></h3>
         <main class="main">
           <article class="doc">
 <h1>OpenShift Serverless Functions</h1>
-
+<div class="paragraph">
+<p>OpenShift Serverless Functions provides a <code>faas</code> CLI that enables developers to create and deploy stateless, event-driven functions as a Knative service on OpenShift Container Platform.</p>
+</div>
+<div class="paragraph">
+<p>The <code>faas</code> CLI uses the CNCF Buildpack API to create a container image.
+Once a container image has been created, the CLI can deploy it as a Knative service on the cluster configured in <code>`~/.kube/config</code>.</p>
+</div>
+<div class="paragraph">
+<p>Buildpacks and function runtimes are currently available for the Go, Node.js and Quarkus languages.</p>
```

Comment:
```suggestion
<p>Serverless Functions supports the Go, Node.js and Quarkus runtimes.</p>
```
Suggesting this, because technically Quarkus and Node.js aren't actually __languages__ they are runtimes for Java and JavaScript respectively.